pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -260,6 +265,8 @@ const (
 // ACMEIssuerDNS01ProviderAkamai is a structure containing the DNS
 // configuration for Akamai DNS—Zone Record Management API
 type ACMEIssuerDNS01ProviderAkamai struct {
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
```

Comment:
I'm not sure if we _need_ a minimum length requirement here... 🤔 I guess it prevents `""` being submitted, but is that already handled by the field being marked required? Not sure how validation works here..